src/controllers/core/controller.js

```diff
@@ -114,4 +114,4 @@ class Controller
         GAME.model.config.money = GAME.model.config.money + parseFloat(amount);
         $("#money").html(GAME.model.config.money.toFixed(2));
     }
-}
+}
```

Comment:
Could you add a single empty line to the end of the file here? After the bracket.